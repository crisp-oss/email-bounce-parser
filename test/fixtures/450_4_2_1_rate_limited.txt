This is the mail system at host mailer.acme.email.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<mulka@gmail.com>: host alt1.gmail-smtp-in.l.google.com[124.215.9.62] said:
    450-4.2.1 The user you are trying to contact is receiving mail at a rate
    that 450-4.2.1 prevents additional messages from being delivered. Please
    resend your 450-4.2.1 message at a later time. If the user is able to
    receive mail at that 450-4.2.1 time, your message will be delivered. For
    more information, please 450-4.2.1 visit 450 4.2.1
    https://support.google.com/mail/?p=ReceivingRate
    f15-20020a056402354f00b00548d4febb0bsi965249edd.614 - gsmtp (in reply to
    RCPT TO command)
