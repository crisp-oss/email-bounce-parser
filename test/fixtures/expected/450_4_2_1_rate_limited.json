{
  "bounce": true,
  "email": {
    "body": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system\n\n<mulka@gmail.com>: host alt1.gmail-smtp-in.l.google.com[124.215.9.62] said: 450-4.2.1 The user you are trying to contact is receiving mail at a rate that 450-4.2.1 prevents additional messages from being delivered. Please resend your 450-4.2.1 message at a later time. If the user is able to receive mail at that 450-4.2.1 time, your message will be delivered. For more information, please 450-4.2.1 visit 450 4.2.1 https://support.google.com/mail/?p=ReceivingRate f15-20020a056402354f00b00548d4febb0bsi965249edd.614 - gsmtp (in reply to RCPT TO command)",
    "intro": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system",
    "error": "<mulka@gmail.com>: host alt1.gmail-smtp-in.l.google.com[124.215.9.62] said: 450-4.2.1 The user you are trying to contact is receiving mail at a rate that 450-4.2.1 prevents additional messages from being delivered. Please resend your 450-4.2.1 message at a later time. If the user is able to receive mail at that 450-4.2.1 time, your message will be delivered. For more information, please 450-4.2.1 visit 450 4.2.1 https://support.google.com/mail/?p=ReceivingRate f15-20020a056402354f00b00548d4febb0bsi965249edd.614 - gsmtp (in reply to RCPT TO command)"
  },
  "data": {
    "error": {
      "code": {
        "basic": "450",
        "enhanced": "4.2.1"
      },
      "label": "The user you are trying to contact is receiving mail at a rate that 450-4.2.1 prevents additional messages from being delivered. Please resend your 450-4.2.1 message at a later time. If the user is able to receive mail at that 450-4.2.1 time, your message will be delivered. For more information, please 450-4.2.1 visit 450 4.2.1 https://support.google.com/mail/?p=ReceivingRate f15-20020a056402354f00b00548d4febb0bsi965249edd.614 - gsmtp",
      "type": "action_not_taken_mailbox_unavailable",
      "temporary": true,
      "permanent": false,
      "data": {
        "type": "action_not_taken_mailbox_rate_limited",
        "blocked": false,
        "spam": false
      }
    },
    "recipient": "mulka@gmail.com",
    "server": {
      "hostname": "alt1.gmail-smtp-in.l.google.com",
      "ip": "124.215.9.62",
      "port": null
    },
    "command": "RCPT TO"
  }
}